doc/startup.md

```diff
@@ -0,0 +1,42 @@
+# Agama Startup Process
+
+This document summarizes how Agama's startup process.
```

Comment:
```suggestion
This document summarizes Agama's startup process.
```

Reply:
Probably off topic, related to what was mentioned on the daily: do we want to mention what happens _before_ Agama starts up? Like, "see Linuxrc and Iguana..."

Reply:
Well, that's more related to agama-live itself. But I will try to come up with something when we have the ALP-based image working.